front/tests/pages/map-model.ts

```diff
@@ -82,6 +82,7 @@ class PlaywrightMap {
       .click();
     await this.page.waitForTimeout(1000);
     await this.page.waitForSelector('.maplibregl-marker');
+    /** click on the center of the map */
```

Comment:
You should put this comment l59, before clickOnMap definition
```suggestion
```

Reply:
done